client/me/help/help-contact-form/index.jsx

```diff
@@ -275,8 +252,12 @@ export const HelpContactForm = React.createClass( {
 
 const mapStateToProps = ( state ) => {
 	return {
-		selectedSiteId: getSelectedSiteId( state )
+		selectedSiteId: getHelpSelectedSiteId( state ) | getSelectedSiteId( state )
```

Comment:
This might actually be the source of this bug I found, where leaving the section and coming back after making a selection causes the dropdown to be empty:

![calypso hc site selector bug](https://user-images.githubusercontent.com/518059/26948623-ad826c54-4c5c-11e7-970f-d74a38f5d5eb.gif)